app/templates/_package.json

```diff
@@ -17,9 +17,9 @@
   "scripts": {
     "flow": "flow",
     "lint": "eslint src",
-    "prepublish": "babel src --out-dir lib",
+    "prepublish": "babel src --out-dir lib --ignore '__tests__/'",
     "jest": "jest src",
-    "test": "npm run flow && npm run lint && npm run jest",
+    "test": "concurrently \"npm run lint\" \"npm run flow\" \"npm run jest\"",
```

Comment:
What about having 2 commands: `test` and `test:concurrency` or maybe some shorthand like `test:c`??
The motivation behind this is that, CI usually use `test` command and it's better to have it running tasks in series, concurrent version should be a opt-in rather than forcing ppl to use that one.